epimodel/exports/epidemics_org.py

```diff
@@ -174,15 +199,26 @@ def extract_models_data(
                 "infected": trace_data.loc[:, "Infected"].tolist(),
                 "recovered": trace_data.loc[:, "Recovered"].tolist(),
             }
+
+            if not np.isinf(initial["Infectious"]):
+                trace["initial_infected"] = initial["Infectious"]
```

Comment:
I think it would be best if we add another value here which is cummulative_active_df[0]. Then we can hopefully faithfully rebuild the whole timeseries using `infected` and `recovered` timeseries.